test/e2e/quadlet/user.container

```diff
@@ -1,7 +1,7 @@
-## assert-podman-final-args imagename
+## assert-podman-final-args localhost/imagename
```

Comment:
The tests currently only exercise "localhost/imagename" for testing a FQN.  Unit tests will certainly increase the coverage but it doesn't hurt testing more here.